projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details-table/query-details-table.component.html

```diff
@@ -0,0 +1,38 @@
+<ng-container *ngIf="queryItem">
+  <table class="govuk-table query-details-table" [attr.aria-describedby]="'Details of the query' | rpxTranslate">
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">{{ 'Last submitted by' | rpxTranslate }}</th>
+        <td class="govuk-table__cell">{{ queryItem.lastSubmittedBy }}</td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">{{ 'Submission date' | rpxTranslate}}</th>
+        <td class="govuk-table__cell">{{ queryItem.createdOn | date: 'd MMM yyyy' }}</td>
```

Comment:
@DavidJayakumar - should the date format not be `dd MMM yyyy`, so that it shows `01 May 2023` instead of `1 May 2023`, for example?